string str
---
geometry_msgs/Pose[] poses